libvirt/install_operator.sh

```diff
@@ -11,11 +11,11 @@ set -o pipefail
 
 script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 
-LIBVIRT_IP="${LIBVIRT_IP:-}"
+LIBVIRT_IP="${LIBVIRT_IP:-192.168.122.1}"
```

Comment:
those seem unrelated. can we move them to their own PR?

Reply:
remove unrelated things, will put them ti a new PR.